src/unordered_lru_cache.h

```diff
@@ -6,6 +6,9 @@
 #define BITCOIN_UNORDERED_LRU_CACHE_H
 
 #include <unordered_map>
+#include <vector>
+#include <cassert>
+#include <algorithm>
```

Comment:
@UdjinM6 This was needed in order to compile after I applied https://github.com/dashpay/dash/pull/4305/files#diff-9cfc75a2c202b215d90e7974267db9e3726d05a8dc6ee835e261f41988da794eL8-R15

And.... I REALLY don't understand why this was needed... Please explain